pkgs/by-name/ws/wslay/package.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, lib, fetchFromGitHub, cmake, cunit }:
+
+stdenv.mkDerivation rec {
+  pname = "wslay";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "tatsuhiro-t";
+    repo = "wslay";
+    rev = "release-${version}";
+    hash = "sha256-xKQGZO5hNzMg+JYKeqOBsu73YO+ucBEOcNhG8iSNYvA=";
+  };
+
+  checkInputs = [ cunit ];
+  doCheck = true;
+
+  nativeBuildInputs = [ cmake ];
```

Comment:
I believe this isn't needed because cmake is added by default. It wasn't needed in my (unreleased) PR at least: https://github.com/NixOS/nixpkgs/pull/170919/files/4fab29665b724d3f6db26b404972ee4623b76919#diff-0830a9b3f789bde4bea0d58c932c6358fdddf725e1346b745e13aa51fe222566

Reply:
It failed when I tried to build the package without it. Honestly, I didn't dig in much to figure out why but this solved the issue.

```
this derivation will be built:
  /nix/store/2r0ar63wa28wdjmgr6wsmsii4rkg2rlm-wslay-1.1.1.drv
building '/nix/store/2r0ar63wa28wdjmgr6wsmsii4rkg2rlm-wslay-1.1.1.drv'...
unpacking sources
unpacking source archive /nix/store/pd4mc63p63fh4pz2w25kh953rkn2v9nf-source
source root is source
patching sources
updateAutotoolsGnuConfigScriptsPhase
configuring
no configure script, doing nothing
building
no Makefile or custom buildPhase, doing nothing
running tests
no Makefile or custom checkPhase, doing nothing
installing
no Makefile or custom installPhase, doing nothing
post-installation fixup
error: builder for '/nix/store/2r0ar63wa28wdjmgr6wsmsii4rkg2rlm-wslay-1.1.1.drv' failed to produce output path for output 'out' at '/nix/store/2r0ar63wa28wdjmgr6wsmsii4rkg2rlm-wslay-1.1.1.drv.chroot/nix/store/sfppd24qm5glin7jgl6d51pwmwp9p7nr-wslay-1.1.1'
```

Reply:
Alright

Reply:
> I believe this isn't needed because cmake is added by default.

No.